### ISSUE TITLE
vidéo streaming

### ISSUE THREAD

**Issue by knobik** [Minims/SomfyProtect2MQTT#100]
dont use bearer token method, just add it as a query parameter :) ` ?access_token=`

**Comment by Minims** [comment]
Good catch :-) Need to find the right path now to retrieve datas.

**Comment by Minims** [comment]
So i found a way to get a ~30s stream. I get a `rtmps://***` URL.
I'm able to convert this stream via
```
ffmpeg -i rtmps://*** -c:v libx264 -c:a aac -ac 1 -strict -2 -crf 18 -profile:v baseline -maxrate 400k -bufsize 1835k -pix_fmt yuv420p -flags -global_header -hls_time 10 -hls_list_size 6 -hls_wrap 10 -start_number 1 streamName.m3u8
```

And then read streamName.m3u8 with VLC. But i stil don't know how to integrate this as the will change every 30s

**Comment by Minims** [comment]
Basic streaming available on dev addons

https://forum.hacf.fr/t/somfyprotect2mqtt/6688/53

**Comment by Minims** [comment]
Streaming beta available in 2023.9.1 
https://github.com/Minims/SomfyProtect2MQTT#video-streaming
